qgis-samgeo-plugin/samgeo_plugin.py

```
@@ -1463,8 +1837,193 @@ def save_masks(self):
+            pass
+
+        # Clear SamGeo model if loaded
+        if self.sam is not None:
+            try:
+                sam_obj = self.sam
+                # Clear the model
+                if hasattr(sam_obj, "model") and sam_obj.model is not None:
+                    try:
+                        sam_obj.model.cpu()
+                    except Exception:  # nosec B110
+                        pass
```

Ruff (0.16.1)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

(SIM105)